experiments/models.py

```diff
@@ -183,6 +185,16 @@ def __str__(self):
     session = models.ForeignKey(
         'ParticipantSession', on_delete=models.CASCADE, null=True)
 
+    @property
+    def status_label(self):
```

Comment:
I'd say that 'status_label' isn't really convention. Often in Django thos would either be called `status` or `get_status_display()`